backend/routes.js

```diff
@@ -130,6 +170,26 @@ export default class Server {
             file: 'oauth.html'
         }));
 
+        server.get(/\/login\/?$/, restify.serveStatic({
+            directory: `${__dirname}/../static`,
+            file: 'login.html'
+        }));
+
+        server.post('/approved/:username', function hasAuth(req, res, next) {
```

Comment:
I kept it as `approve` though. I don't think showing the list of users is a great idea.. Just checking if a provided username is approved is good I think.